helixbass/tree-sitter-lint-plugin-eslint-builtin
# Expose "all" config

In this PR:
- expose `all` config which turns on all currently-implemented rules

To test:
You should be able to put `extends: [eslint-builtin/all]` in a config file and see it run all of the currently implemented lint rules
In theory you should be able to then override those by eg manually setting some of those rules to `level: off` in the `rules` section of the config file
